### DISCUSSION TITLE
For version greater than `19.3.3`, autobahn is not able to run in background thread

### DISCUSSION THREAD

**Discussion by protojimmylin** [crossbario/autobahn-python#1624] [Announcements]
Are there any solutions/workarounds for this?

**Comment by oberstet** [comment]
well,, this is not true - autobahn is able to run user code on background threads, e.g. publish a WAMP event using `callFromThread` (https://docs.twisted.org/en/stable/core/howto/threading.html) for Twisted and so on ..

**Reply by protojimmylin** [reply]
Thank you. I'll look closely into the docs.
